IPython/core/interactiveshell.py

```diff
@@ -2613,7 +2615,7 @@ def run_cell(self, raw_cell, store_history=False, silent=False, shell_futures=Tr
         result = ExecutionResult()
 
         if (not raw_cell) or raw_cell.isspace():
-            self.last_execution_succeeded = True
+            self.last_execution_succeeded, self.last_execution_result = True, result
```

Comment:
There's no need to resubmit - just push more commits to the same branch, the pull request will update and the tests will re-run. :-)